debug_toolbar/apps.py

```diff
@@ -208,7 +208,29 @@ def js_mimetype_check(app_configs, **kwargs):
     return []
 
 
-@register()
+@register
+def debug_toolbar_installed_when_running_tests_check(app_configs, **kwargs):
+    """
+    Check that the toolbar is not being used when tests are running
+    """
+    if not settings.DEBUG and dt_settings.get_config()["IS_RUNNING_TESTS"]:
+        return [
+            Error(
+                "The Django Debug Toolbar can't be used with tests",
+                hint="Django changes the DEBUG setting to False when running "
+                "tests. By default the Django Debug Toolbar is installed because "
+                "DEBUG is set to True. For most cases, you need to avoid installing "
+                "the toolbar when running tests. If you feel this check is in error, "
+                "you can set `DEBUG_TOOLBAR_CONFIG['IS_RUNNING_TESTS'] = False` to "
+                "bypass this check.",
+                id="debug_toolbar.W009",
```

Comment:
I notice that we're using a different naming convention to Django here. Django uses `W` for warnings and `E` for errors. Just saying, it doesn't have to be changed.

Reply:
That's a good catch. We should follow that standard.

Reply:
I updated it to E001